addons/mail/static/src/components/discuss/discuss_tests.js

```diff
@@ -559,6 +559,28 @@ QUnit.test('sidebar: channel rendering with needaction counter', async function
     );
 });
 
+QUnit.test('sidebar: channel rendering with mass mailing', async function (assert) {
+    assert.expect(1);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: "channel",
+                id: 20,
+                mass_mailing: true,
+                name: "General",
+            }],
+        },
+    });
+    await this.start();
+    const channel = document.querySelector(`.o_DiscussSidebar_groupChannel .o_DiscussSidebar_item`);
+    assert.containsOnce(
+        channel,
+        '.fa.fa-envelope-o',
+        "should have an icon to indicate that the channel is mass mailing"
```

Comment:
=> `... is a mailing channel` (let's avoid using term "mass mailing" when we can)